### PR TITLE
Freight cleanups

### PR DESCRIPTION
**Pull request by ekohl**
See individual commits for details

### CODE REVIEW

File: puppet/modules/freight/manifests/install.pp [hunk 1]
```diff
@@ -3,7 +3,7 @@
   if $::operatingsystem == 'Debian' {
     apt::source { 'freight':
       location    => 'http://packages.rcrowley.org',
```
Comment (mmoll) [inline puppet/modules/freight/manifests/install.pp]: https://github.com/freight-team/freight#from-a-debian-archive would be used these days on Debian/Ubuntu. ;)

File: puppet/modules/freight/manifests/install.pp [hunk 2]
```diff
@@ -3,7 +3,7 @@
   if $::operatingsystem == 'Debian' {
```
Comment (mmoll) [inline puppet/modules/freight/manifests/install.pp]: osfamily is better now

**Comment by ekohl** [comment]
Updated

File: puppet/modules/freight/manifests/install.pp [hunk 3]
```diff
@@ -2,11 +2,11 @@
 
   if $::osfamily == 'Debian' {
     apt::source { 'freight':
-      location    => 'http://packages.rcrowley.org',
+      location    => 'http://build.openvpn.net/debian/freight_team',
       release     => $::lsbdistcodename,
       repos       => 'main',
       key         => '7DF49CEF',
```
Comment (mmoll) [inline puppet/modules/freight/manifests/install.pp]: that ID won't be the same ;)

Comment (ekohl) [inline puppet/modules/freight/manifests/install.pp]: Fixed. Looks like it'll expire in 2 months though.

**Comment by mmoll** [comment]
merged, bedankt @ekohl!

**Pull request merged**